docs/rfcs/active/dashboards/1778-dashboards.md

```diff
@@ -0,0 +1,117 @@
+`Dashboard_A` and `Chart_3` and `Chart_4` are part of `Dashboard_B`.
+
+### Entities
+There will be 2 top level GMA [entities](../../../what/entity.md) in the design: dashboards and charts.
+It's important to make charts as a top level entity because charts could be shared between different dashboards.
+We'll need to build `Contains` relationships between Dashboard and Chart entities.
+
+### URN Representation
+We'll define two [URNs](../../../what/urn.md): `DashboardUrn` and `ChartUrn`.
+These URNs should allow for unique identification for dashboards and charts even there are multiple dashboarding tools
+are used within a company. Most of the time, dashboards & charts are given unique ids by the used dashboarding tool.
+An example Dashboard URN for Looker will look like below:
+```
+urn:li:dashboard:(Looker,<<dashboard_id>>)
+```
+An example Chart URN for Redash will look like below:
+```
+urn:li:chart:(Redash,<<chart_id>>)
+```
+
+### Chart metadata
+Dashboarding tools generally have different jargon to denote a chart.
+They are called as [Look](https://docs.looker.com/exploring-data/saving-and-editing-looks) in Looker 
+and [Visualization](https://redash.io/help/user-guide/visualizations/visualization-types) in Redash.
+But, irrespective of the name, charts are the different tiles which exists in a dashboard.
+Charts are mainly used for delivering some information in a visual way to make it easily understandable.
+They might be using single or multiple data sources and generally have an associated query running against
+the underlying data source to generate the data that it will present.
+
+Below is a list of metadata which can be associated with a chart:
+
+- Title
+- Description
+- Type (Bar chart, Pie chart, Scatter plot etc.)
+- Input sources
+- Query (and its type)
+- Access level (public, private etc.)
+- Ownership
+- Status (removed or not)
+- Audit info (last modified, last refreshed)
+
+### Dashboard metadata
+Aside from containing a set of charts, dashboards carry metadata attached to them.
+Below is a list of metadata which can be associated with a dashboard:
+
+- Title
+- Description
+- List of charts
+- Access level (public, private etc.)
+- Ownership
+- Status (removed or not)
+- Audit info (last modified, last refreshed)
+
+### Metadata graph
+
+![dashboards_graph](imgs/dashboards_graph.png)
+
+An example metadata graph showing complete data lineage picture is shown above.
+In this picture, `Dash_A` and `Dash_B` are dashboards and they are connected to charts through `Contains` edges.
+`C1`, `C2`, `C3` and `C4` are charts and they are connected to underlying datasets through `DownstreamOf` edges.
+`D1`, `D2` and `D3` are datasets.
+
+## How we teach this
+
+We should create/update user guides to educate users for:
+ - Search & discovery experience (how to find a dashboard in DataHub)
+ - Lineage experience (how to find upstream datasets of a dashboard and how to find dashboards generated from a dataset)
+
```

Comment:
Also include example crawlers

Reply:
Didn't quite get this. You mean crawlers as a separate section or into the "how we teach this" section?

Reply:
I mean education the users through example crawlers.

Reply:
My initial understanding was that this section is mainly to answer `How we teach this to end users of DataHub` not the `developers`? Example crawler guide sounds like a developer guide to me? @jplaisted Can you clarify what this section aims to cover?

Reply:
Ping @jplaisted for clarification on this.